static/javascripts/components/profile.jsx

```diff
@@ -66,25 +60,21 @@ getInitialState: function(){
 },
 
     render: function(){
-    console.log(user_data)
-    return(
-        <div className="mdl-card mdl-shadow--2dp">
```

Comment:
Yes, I didn't think the card looked good. After I made the change though, I saw that you had recently created it. We can do it either way. There's no technical need for the information to be in a card, so it's all about how we want to present it.